app/src/main/java/com/otaman/weather/ui/MainScreen.kt

```diff
@@ -0,0 +1,302 @@
+            )
+            Text(
+                text = "Cloudy",
+                style = MaterialTheme.typography.h3
+            )
+            Spacer(modifier = Modifier.padding(12.dp))
+            Row(
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_wind),
+                    contentDescription = null
+                )
+                Text(
+                    text = "Wind",
+                    style = MaterialTheme.typography.h1,
+                    textAlign = TextAlign.Center,
+                    fontSize = 18.sp,
+                    modifier = Modifier
+                        .width(80.dp)
+                )
+                Text(
+                    text = "|",
+                    style = MaterialTheme.typography.h1,
+                    textAlign = TextAlign.Center,
+                    fontSize = 12.sp,
+                    modifier = Modifier.width(40.dp)
+                )
+                Text(
+                    text = "10 km/h",
+                    style = MaterialTheme.typography.h1,
+                    textAlign = TextAlign.Start,
+                    fontSize = 18.sp,
+                    modifier = Modifier
+                        .width(90.dp)
+                        .padding(start = 20.dp)
+                )
+            }
+            Spacer(modifier = Modifier.padding(6.dp))
+            Row(
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_hum),
+                    contentDescription = null
+                )
+                Text(
+                    text = "Hum",
+                    style = MaterialTheme.typography.h1,
+                    textAlign = TextAlign.Center,
+                    fontSize = 18.sp,
+                    modifier = Modifier.width(80.dp)
+                )
+                Text(
+                    text = "|",
+                    style = MaterialTheme.typography.h1,
+                    textAlign = TextAlign.Center,
+                    fontSize = 12.sp,
+                    modifier = Modifier.width(40.dp)
+                )
+                Text(
+                    text = "54 %",
+                    style = MaterialTheme.typography.h1,
+                    textAlign = TextAlign.Start,
+                    fontSize = 18.sp,
+                    modifier = Modifier
+                        .width(90.dp)
+                        .padding(start = 20.dp)
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun WeatherContent() {
+    Column(
+        modifier = Modifier
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        WeatherType()
+        WeatherDetail()
+    }
+}
+
+@Composable
+fun WeatherApp() {
+    WeatherTheme {
```

Comment:
Few suggestions:
1. Rename `WeatherApp` to something like `MainScreen`
2. Remove `WeatherTheme` because it's redundant (already wrapped in `MainActivity`)